Add User::inAnyRole method

8633brown opened the pull request:
#440

brunogaspar commented on src/Users/EloquentUser.php:
```diff
@@ -220,6 +220,25 @@ public function inRole($role)
+            return $this->inRole($roles);
+        }
+
+        return false;
```
Just wondering if this should move to line `234` instead? As it seems this would only be hit if the loop ends without any results?

8633brown replied:
I was keeping to the same structure as the [inRole](https://github.com/cartalyst/sentinel/blob/36d802f962dc14f7c0639a94976892fa0708dc23/src/Users/EloquentUser.php#L220) method.

brunogaspar replied:
It's all good, no worries.

Will merge tomorrow and tag it, thanks!

8633brown replied:
So I was also just copying the code from the issue linked. From what I understand changing where you return from in this situation is opinion based. I'm happy to change it. I'm just wondering if theres any more reasoning behind keeping the return within the ```if``` statement and keeping it at the end of the method.

brunogaspar replied:
It's all good @8633brown :)

brunogaspar commented:
@8633brown Unfortunately, i'll need to make this merged into the next release due to the fact that adding new methods is somewhat a breaking change according to SemVer.

Don't think you'll need to do anything, i just need to sync `master` with `2.0` and switch the target branch on this pull request so i can merge it, hopefully heh

8633brown commented:
Switched branches for you to merge with master. Resolved one conflict which was just an extra empty line and a comment. 

This repo was up to date with 2.0 when I added the new method. So it has all of the new changes in it. I guess it still makes sense to merge 2.0 into master.

brunogaspar commented:
I've synced `master` with `2.0`, can you please do a rebase @8633brown so it removes the unnecessary commits/changes?

8633brown commented:
should be all sorted. had some trouble with a rebase. I think master was merged into my repo when i edited the PR. so i pullud then rebased then pushed which messed it all up i think. so had to reset it back to before the merge with master and force push it.
sorry bit of a messy way of doing things but gotta learn somehow.

brunogaspar commented:
> should be all sorted. had some trouble with a rebase. I think master was merged into my repo when i edited the PR. so i pullud then rebased then pushed which messed it all up i think. so had to reset it back to before the merge with master and force push it.

> sorry bit of a messy way of doing things but gotta learn somehow.

heh no worries, that's the best way to learn :)

Thanks for the feature, appreciate it!
